ui/pages/settings/settings.component.js

```diff
@@ -78,17 +78,6 @@ class SettingsPage extends PureComponent {
     searchText: '',
   };
 
-  shouldRenderExperimentalTab() {
-    ///: BEGIN:ONLY_INCLUDE_IN(desktop)
-    const desktopAvailable = true;
-    if (desktopAvailable) {
-      return true;
-    }
-    ///: END:ONLY_INCLUDE_IN
-
-    return process.env.TRANSACTION_SECURITY_PROVIDER || process.env.NFTS_V1;
-  }
-
```

Comment:
I think 

```
///: BEGIN:ONLY_INCLUDE_IN(desktop)
    const desktopAvailable = true;
    if (desktopAvailable) {
      return true;
    }
    ///: END:ONLY_INCLUDE_IN
```
Is important and experimental tab should be rendered in desktop app.

Reply:
I see what you mean, but with this PR `experimental` tab will be shown to the users by default in any case. The reason for its removal is the fact that just removing the feature flag for the `TRANSACTION_SECURITY_PROVIDER` won't show the `OpenSea security provider` in the `experimental` tab, unless this conditional rendering gets removed.